docs/eventing/samples/kafka/source/README.md

```diff
@@ -0,0 +1,202 @@
+   event-display-00001-deployment-5d5df6c7-gv2j4   2/2       Running   0          72s
+   ...
+   ```
+
+#### Apache Kafka Event Source
+
+1. Modify `source/event-source.yaml` accordingly with bootstrap
+   servers, topics, etc...:
+
+   ```yaml
+   apiVersion: sources.eventing.knative.dev/v1alpha1
+   kind: KafkaSource
+   metadata:
+     name: kafka-source
+   spec:
+     consumerGroup: knative-group
+     bootstrapServers: my-cluster-kafka-bootstrap.kafka:9092 #note the kafka namespace
+     topics: knative-demo-topic
+     sink:
+       apiVersion: serving.knative.dev/v1alpha1
+       kind: Service
+       name: event-display
+   ```
+
+1. Deploy the event source.
+   ```
+   $ kubectl apply -f kafka/source/samples/event-source.yaml
+   ...
+   kafkasource.sources.eventing.knative.dev/kafka-source created
+   ```
+1. Check that the event source pod is running. The pod name will be prefixed
+   with `kafka-source`.
+   ```
+   $ kubectl get pods
+   NAME                                  READY     STATUS    RESTARTS   AGE
+   kafka-source-xlnhq-5544766765-dnl5s   1/1       Running   0          40m
+   ```
+1. Ensure the Apache Kafka Event Source started with the necessary
+   configuration.
+   ```
+   $ kubectl logs --selector='knative-eventing-source-name=kafka-source'
+   {"level":"info","ts":"2019-04-01T19:09:32.164Z","caller":"receive_adapter/main.go:97","msg":"Starting Apache Kafka Receive Adapter...","Bootstrap Server":"...","Topics":".","ConsumerGroup":"...","SinkURI":"...","TLS":false}
+   ```
+
+### Verify
+
+1. Produce a message (`{"msg": "This is a test!"}`) to the Apache Kafka topic, like shown below:
+   ```
+   kubectl -n kafka run kafka-producer -ti --image=strimzi/kafka:0.14.0-kafka-2.3.0 --rm=true --restart=Never -- bin/kafka-console-producer.sh --broker-list my-cluster-kafka-bootstrap:9092 --topic my-topic
+
+
+  If you don't see a command prompt, try pressing enter.
+  >{"msg": "This is a test!"}
+   ```
+1. Check that the Apache Kafka Event Source consumed the message and sent it to
+   its sink properly.
+
+   ```
+   $ kubectl logs --selector='knative-eventing-source-name=kafka-source'
+   ...
+   {"level":"info","ts":"2019-04-15T20:37:24.702Z","caller":"receive_adapter/main.go:99","msg":"Starting Apache Kafka Receive Adapter...","bootstrap_server":"...","Topics":"knative-demo-topic","ConsumerGroup":"knative-group","SinkURI":"...","TLS":false}
+   {"level":"info","ts":"2019-04-15T20:37:24.702Z","caller":"adapter/adapter.go:100","msg":"Starting with config: ","bootstrap_server":"...","Topics":"knative-demo-topic","ConsumerGroup":"knative-group","SinkURI":"...","TLS":false}
+   {"level":"info","ts":1553034726.546107,"caller":"adapter/adapter.go:154","msg":"Successfully sent event to sink"}
+   ```
+
+1. Ensure the Event Display received the message sent to it by the Event Source.
+
+   ```
+   $ kubectl logs --selector='serving.knative.dev/service: event-display' -c user-container
```

Comment:
Oops now that I'm looking at this again, I see it was the wrong syntax. Label selectors use `=` as a separator. Sorry about that :disappointed:

`--selector='serving.knative.dev/service=event-display'`